packages/frontend/src/features/catalog/components/CatalogFieldListItem.tsx

```diff
@@ -22,40 +22,58 @@ export const CatalogFieldListItem: FC<React.PropsWithChildren<Props>> = ({
 
     return (
         <>
-            <Group
-                noWrap
-                py="xs"
+            <UnstyledButton
```

Comment:
Why button? So you get focus and tabbing? The focus conflicts with our custom keyboard nav a bit, but maybe it's fine. If we want to put anything else interactive in these, it's going to need to become something else, but so far we dont.

Reply:
Good point! Let me change it back and see if things remain the same 👍 
My only reason for this was to make it an interactive element, but I understand the concern 💪

Reply:
here! https://github.com/lightdash/lightdash/pull/10255/commits/24dbc17fa7055306df9029fa16f2bfd62bd80a5f